x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/constants.ts

```diff
@@ -197,6 +197,6 @@ export const getConnectorTemplate = ({
 }
 
   elasticsearch:
-    host: "${host || 'https://locahost:9200'}"
+    host: "${host || 'https://localhost:9200'}"
```

Comment:
Actual question because I don't know: should it be `http` instead of `https`?

Reply:
This should be `http`!